src/karma-webpack.js

```diff
@@ -18,6 +18,18 @@ let isBlocked = false;
 
 const normalize = (file) => file.replace(/\\/g, '/');
 
+var getOutputPath = (outputPath) => {
```

Comment:
Can we make this just a function instead? Or use `const`
```suggestion
function getOutputPath(outputPath) {
```
```suggestion
const getOutputPath = (outputPath) => {
```